integration-cli/check_test.go

```diff
@@ -33,6 +33,7 @@ func (s *DockerSuite) TearDownTest(c *check.C) {
 	deleteAllImages()
 	deleteAllVolumes()
 	deleteAllNetworks()
+	cleanupShm()
```

Comment:
My bad, just checked go-check code.  It actually handles defer. I'll update my patch.